From 8802bad31a37ded724cf157e49b6491226654d35 Mon Sep 17 00:00:00 2001
From: Ali Can Ozaslan <ali.oezaslan@arm.com>
Date: Tue, 29 Apr 2025 07:52:14 +0000
Subject: [PATCH] Align PSA Crypto structs with TF-Mv2.1.1

The psa_client_key_attributes_s struct had to be aligned with the
psa_key_attributes_s struct in TF-M.

Change-Id: Ia3b5e8f313bd79a321dda6a61357c1a5e5a17bc2
Signed-off-by: Ali Can Ozaslan <ali.oezaslan@arm.com>
Signed-off-by: Musa Antike <musa.antike@arm.com>
Signed-off-by: Frazer Carsley <frazer.carsley@arm.com>

Upstream-Status: Backport [8802bad31a37ded724cf157e49b6491226654d35 on integration branch]
---
 components/service/crypto/include/psa/crypto_client_struct.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/components/service/crypto/include/psa/crypto_client_struct.h b/components/service/crypto/include/psa/crypto_client_struct.h
index 7e907faa..db2f9161 100644
--- a/components/service/crypto/include/psa/crypto_client_struct.h
+++ b/components/service/crypto/include/psa/crypto_client_struct.h
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020-2023, Arm Limited. All rights reserved.
+ * Copyright (c) 2020-2025, Arm Limited. All rights reserved.
  *
  * SPDX-License-Identifier: BSD-3-Clause
  *
@@ -38,7 +38,6 @@ struct psa_client_key_attributes_s
     uint32_t alg;
     uint32_t alg2;
     uint32_t id;
-    int32_t owner_id;
 };
 
 #define PSA_CLIENT_KEY_ATTRIBUTES_INIT {0}
-- 
2.43.0

